client/jetpack-cloud/sections/sidebar-navigation/jetpack-manage.tsx

```diff
@@ -1,12 +1,63 @@
+import { plugins, key, currencyDollar, category } from '@wordpress/icons';
+import { useTranslate } from 'i18n-calypso';
+import page from 'page';
 import NewSidebar from 'calypso/jetpack-cloud/components/sidebar';
+import { itemLinkMatches } from 'calypso/my-sites/sidebar/utils';
+import {
+	JETPACK_MANAGE_DASHBOARD_LINK,
+	JETPACK_MANAGE_PLUGINS_LINK,
+	JETPACK_MANAGE_LICENCES_LINK,
+	JETPACK_MANAGE_BILLING_LINK,
+} from './lib/constants';
 
-// This sidebar is what Jetpack Manage customers will see by default as they
-// navigate around Jetpack Cloud with no specific site selected.
-// It'll display menu options like Sites Management, Plugin Management,
-// and Purchases.
-// FIXME: Add menu items
 const JetpackManageSidebar = () => {
-	return <NewSidebar isJetpackManage path="/" menuItems={ [] } />;
+	const translate = useTranslate();
+
+	const onClickMenuItem = ( path: string ) => {
+		page.redirect( path );
+	};
+
+	const isSelected = ( link: string ) => {
+		const pathname = window.location.pathname;
+		return itemLinkMatches( link, pathname );
+	};
```

Comment:
[Non-blocking, performance]

Both these functions can be moved outside/above the component, because they don't depend on any part of the component's internal state (or even React). This will prevent them from being recreated on every render (though I don't expect this component will re-render often at all).